Firmware/AnalogInput/app_funcs.c

```diff
@@ -312,6 +314,12 @@ void app_read_REG_DO0_PULSE(void)
 bool app_write_REG_DO0_PULSE(void *a)
 {
 	uint8_t reg = *((uint8_t*)a);
+	
+	if (reg < 1)
```

Comment:
The minimum value in the schema should also be updated if I understand it correctly.

Reply:
The minimum value was already 1. I don't think any change is needed.